<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="17484" started-at="2017-04-25T18:53:48Z" finished-at="2017-04-25T18:54:05Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="17484" started-at="2017-04-25T18:53:48Z" finished-at="2017-04-25T18:54:05Z">
      <class name="testi_Sele_testNG.testi_Sele_testNG">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testi_Sele_testNG.testi_Sele_testNG@7dc7cbad]" name="beforeMethod" is-config="true" duration-ms="1848" started-at="2017-04-25T21:53:48Z" finished-at="2017-04-25T21:53:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="main()[pri:0, instance:testi_Sele_testNG.testi_Sele_testNG@7dc7cbad]" name="main" duration-ms="15497" started-at="2017-04-25T21:53:50Z" finished-at="2017-04-25T21:54:05Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.remote.RemoteWebElement.isEnabled(RemoteWebElement.java:149)
	at org.openqa.selenium.support.ui.ExpectedConditions$24.apply(ExpectedConditions.java:703)
	at org.openqa.selenium.support.ui.ExpectedConditions$24.apply(ExpectedConditions.java:698)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	at testi_Sele_testNG.testi_Sele_testNG.main(testi_Sele_testNG.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testi_Sele_testNG.testi_Sele_testNG@7dc7cbad]" name="afterMethod" is-config="true" duration-ms="95" started-at="2017-04-25T21:54:05Z" finished-at="2017-04-25T21:54:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testi_Sele_testNG.testi_Sele_testNG -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
